oasislmf/cli/exposure.py

```diff
@@ -115,6 +119,8 @@ def action(self, args):
                 'Invalid output level. Must be one of port, acc, loc, pol or item.'
             )
 
+        output_file = inputs.get('output_file', default=None, required=False)
```

Comment:
@mpinkerton-oasis Since `output_file` is a path we need to apply a little method we use everywhere else in the CLI when dealing with file paths, which is `as_path` (a method in `utils.path` which is already imported in `exposure.py`). This method is used to do a couple of things: convert relative paths to absolute paths if the original argument was provided as a relative path, and also check whether the paths pre-exist or are created in the course of the backend execution. Here `output_file` appears to be optional and doesn't necessarily exist prior to running the command. So I would replace line 122 with this one

    as_path(inputs.get('output_file', required=False, is_path=True), 'Output file path', preexists=False)

Reply:
Fixed